### DISCUSSION TITLE
Keyclick repeat rate issue

### DISCUSSION THREAD

**Discussion by 07kailash** [freeplane/freeplane#2099] [Help]
Dear community,
I've been using and enjoying freeplane a lot lately - great tool, many thanks!
(FP 1.12.6, Win10 Pro)
Am using keyboard shortcuts mostly (ALT+key), but auto-repeat of the keyclick kicks in very fast, often overwriting core text of the current node with the repeated key. If unnoticed, this leads to unwanted data loss.
Have adjusted the OI keyboard settings to longest delay and moderate repeat rate - but seems to have no effect on FP.
All suggestions and help is very welcome, thanks in advance

**Comment by 07kailash** [comment]
might add:
- I meant **OS** keyboard settings
- this "double key-entry" issue also pertains to the space bar when folding/unfolding nodes

**Comment by i-plasm** [comment]
Hi @07kailash, you might find this helpful as a partial workaround: https://github.com/freeplane/freeplane/issues/116#issuecomment-2483877018

**Reply by 07kailash** [reply]
Thx @i-plasm , for the _partial_ workaround, as bitclick mentioned, we have to trade for speed then.
IMHO the question on key repeat rate remains ...

**Reply by i-plasm** [reply]
I suspect the problem does not have to do with Freeplane. It's probably something about your OS configuration and/or hardware, drivers, etc... Best regards.
